config/locales/en/coronavirus_landing_page.yml

```diff
@@ -3,6 +3,17 @@ en:
     meta_title: "Coronavirus (COVID-19): what you need to do"
     meta_description: "Find out about the government response to coronavirus (COVID-19) and what you need to do."
     page_header: "Coronavirus (COVID-19): what you need to do"
+    stay_at_home:
+      pretext: "Stay at home:"
```

Comment:
This doesn't match the content doc. Maybe need to check with ben. Content doc has:
`Stay at home. Protect the NHS. Save lives.`
If it's meant to match the design then need to lose the `:`

Reply:
have asked Ben

Reply:
Ok ignore - sam confirmed it's just mean to be Stay at home, so just need to lost the `:`

Reply:
Done